oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/celliterator/CellIterator.java

```diff
@@ -49,6 +48,8 @@
 import org.locationtech.jts.geom.TopologyException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Streams;
```

Comment:
odd… those imports should have remained at the top (before `import java.io.Serializable;`).

Reply:
(similar issues exist also in other classes, .g. `Transform*.java`)

Reply:
so I reverted the reordering of the import statement. Is there still something missing before we can merge this PR?

Reply:
at first glance no, but a code review is definitely still missing :stuck_out_tongue_closed_eyes: can you remove the "draft" status, then I'll take a look.